### PR TITLE
Add a Reset to Timer FBs

### PR DESCRIPTION
**Pull request by franz-hoepfinger-4diac**
solves https://github.com/eclipse-4diac/4diac-ide/issues/78

### CODE REVIEW

**Comment by diplfranzhoepfinger** [comment]
@azoitl any help needed ? 

eclipse-4diac/4diac-ide#79 was merged. Thanks so far.

**Comment by azoitl** [comment]
> @azoitl any help needed ? 
> 
> [eclipse-4diac/4diac-ide#79](https://github.com/eclipse-4diac/4diac-ide/pull/79) was merged. Thanks so far.

@franz-hoepfinger-4diac oh sorry I thought I already had reviewed and merged it. Will do now.

**Review by azoitl — APPROVED** [review]

File: src/stdfblib/events/E_TP_fbt.cpp
Comment (azoitl) [inline src/stdfblib/events/E_TP_fbt.cpp:1]: The reset event and the according event connections are not in this file

File: src/stdfblib/events/E_TP_fbt.h
Comment (azoitl) [inline src/stdfblib/events/E_TP_fbt.h:1]: The reset event and the according event connections are not in this file

File: src/stdfblib/events/E_PULSE_fbt.h
Comment (azoitl) [inline src/stdfblib/events/E_PULSE_fbt.h:1]: The reset event and the according event connections are not in this file

File: src/stdfblib/events/E_PULSE_fbt.cpp
Comment (azoitl) [inline src/stdfblib/events/E_PULSE_fbt.cpp:1]: The reset event and the according event connections are not in this file

**Review by azoitl — CHANGES_REQUESTED** [review]
I was to fast with my approval. I think there are some issues with this PR.

**Comment by franz-hoepfinger-4diac** [comment]
> I was to fast with my approval. I think there are some issues with this PR.

what is missing ? 
it is auto-Generated Code, so it should be 1:1 what the IDE made ... 

i just checked it again, also the `g_nStringIdR` is there.

**Comment by azoitl** [comment]
I thought compared to the XML file changes more changes should be in the cpp. But as you pointed out they are already there. Sorry.

**Review by azoitl — APPROVED** [review]

**Comment by diplfranzhoepfinger** [comment]
thanks. !

**Pull request merged**